### PR TITLE
docs: add docs

### PR DESCRIPTION
**Pull request by xzchaoo**
# Which issue does this PR close?

# Rationale for this change
Add docs.

# What changes are included in this PR?
Add docs.

# Are there any user-facing changes?

# How does this change test
I test mdbook's render result locally using 'mdbook serve'.

I test 'Github Pages' render result using my fork repository, and deploy pages to https://xzchaoo.github.io/holoinsight-docs/ .

### CODE REVIEW

File: docs/en/src/dev-guide/ddl-management.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# DDL management
+
+1. 尽可能提前创建 issue, 然后才开始写代码
```
Comment (xiangwanpeng) [inline docs/en/src/dev-guide/ddl-management.md]: Chinese appears in `docs/en`.

File: docs/en/src/dev-guide/ddl-management.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+2. 对 issue 的最低要求: 最起码像英语, 没有明显的语法错误, 实在不行 Google 翻译转一下.
+3. 该讨论的点需要在 issue/PR 上讨论一下, 不要私聊, 合并不要变成走流程了
+4. 新增的改动的代码的注释必须全英文, 你要是愿意可以顺便留一份中文的翻译; 各位 IDEA 创建Java文件时的模板记得改掉
+5. 几个地方必须写注释
+   1. *所有* 公开的类/接口
```
Comment (jsy1001de) [inline docs/en/src/dev-guide/ddl-management.md]: here is the english directory，need english description

**Review by xiangwanpeng — CHANGES_REQUESTED** [review]

File: docs/zh/src/dev-guide/build/build-docker-image.md [hunk 3]
```diff
@@ -0,0 +1,195 @@
+# 构建 server docker 镜像
+
+## 构建多架构 docker 镜像
+```bash
+sh ./scripts/docker/buildx.sh
```
Comment (jiwliu) [inline docs/zh/src/dev-guide/build/build-docker-image.md:5]: buildx.sh  does not exist

File: docs/en/src/dev-guide/build/docker-image-details.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+Here are some details of Dockerfile:
+1. Install azul openjdk at /opt/java8
+2. Use [Supervisord](http://supervisord.org/introduction.html) to control our Java and Nginx processes.
+3. Java/Nginx worker processes run as user 'admin'
+4. The app fat jar location is /home/admin/app.jar
+5. The log location is `/home/admin/logs/holoinsight-server/`
+6. The front-end resources location is `/home/admin/holoinsight-server-static/`
+7. `alias g="cd /home/admin/logs/holoinsight-server/"` 
+8. The helper scripts location is `/home/admin/logs/api/` which is a soft link to `/home/admin/api/`. Considering that most of the time I will be in the `/home/admin/logs/holoinsight-server/` directory (using command 'g'), I put the api directory here and use syntax `sh ../api/basic/version` refers to them
```
Comment (xiangwanpeng) [inline docs/en/src/dev-guide/build/docker-image-details.md]: Avoiding first-person pronouns like "I" may be more intelligible to the reader.

File: docs/zh/src/dev-guide/build/build-docker-image.md (hunk 3 above)
Comment (xzchaoo) [reply docs/zh/src/dev-guide/build/build-docker-image.md:5]: https://github.com/traas-stack/holoinsight/blob/main/scripts/docker/buildx.sh

File: docs/en/src/dev-guide/build/docker-image-details.md (hunk 4 above)
Comment (xzchaoo) [reply docs/en/src/dev-guide/build/docker-image-details.md]: OK. The docs now use 'We' pronoun.

**Review by jsy1001de — APPROVED** [review]
LGTM

**Review by xiangwanpeng — APPROVED** [review]

**Pull request merged**